### PR TITLE
i18n(hi): translate`why-astro.mdx`, `install/manual.mdx`, `editor-setup.mdx` & `nav.ts`

### PR DESCRIPTION
**Pull request by at-the-vr**
#### Description (required)
- updates translations for [why-astro](https://github.com/withastro/docs/commits/main/src/content/docs/en/concepts/why-astro.mdx?since=2023-10-23T20:05:26.000Z), [editor-setup](https://github.com/withastro/docs/commits/main/src/content/docs/en/editor-setup.mdx?since=2024-01-18T11:34:16.000Z), [install/manual](https://github.com/withastro/docs/commits/main/src/content/docs/en/install/manual.mdx?since=2024-01-18T11:34:16.000Z) and [nav.ts](https://github.com/withastro/docs/commits/main/src/i18n/en/nav.ts?since=2023-12-26T12:40:04.000Z) in Hindi Language

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/i18n/hi/nav.ts [hunk 1]
```diff
@@ -74,4 +74,8 @@ export default NavDictionary({
 	'reference/cli-reference': 'Astro CLI',
 	'reference/error-reference': 'त्रुटि संदर्भ',
 	'guides/publish-to-npm': 'NPM पैकेज प्रारूप',
+
+	communityResources: 'सामूहिक संसाधन',
+	'community-resources/content': 'पाठ्यक्रम, मार्गदर्शिकाएँ और रेसिपी',
+	'community-resources/talks': 'वार्ता, साक्षात्कार और स्ट्रीमस',
```
Comment (Mayank-Malviya) [inline src/i18n/hi/nav.ts:80]:
```suggestion
	'community-resources/talks': 'वार्ता, इंटरव्यू, और स्ट्रीमस',
```

File: src/content/docs/hi/concepts/why-astro.mdx [hunk 2]
```diff
@@ -1,45 +1,57 @@
...
 ---
 
-Astro **तेज़**, **सामग्री-केंद्रित वेबसाइटें** बनाने के लिए एक **ऑल-इन-वन वेब फ्रेमवर्क** है।
+**Astro** ब्लॉग, मार्केटिंग और ई-कॉमर्स जैसी **सामग्री-संचालित वेबसाइटें** बनाने के लिए वेब फ्रेमवर्क है। Astro को अन्य फ्रेमवर्क की तुलना में JavaScript ओवरहेड और जटिलता को कम करने के लिए एक नयी [फ्रंटएंड वास्तुकला](/hi/concepts/islands/) का नेतृत्व करने के लिए जाना जाता है। यदि आपको ऐसी वेबसाइट चाहिए जो तेजी से लोड हो और उसका SEO बढ़िया हो, तो Astro आपके लिए है।
 
-किसी अन्य वेब फ्रेमवर्क के बजाय Astro को क्यों चुनना चाहिए ? यहां पांच आन्तरक डिज़ाइन सिद्धांत दिए गए हैं जो आपको यह समझाने में मदद करेंगे हमने Astro को क्यों बनाया, वे समस्याएँ जिनका समाधान करने के लिए यह मौजूद हैं, एवं Astro आपके परियोजना या आपकी टीम के लिए सबसे उत्तम विकल्प क्यों हो सकता है।
+## विशेषताएँ
 
-#### Astro is...
+**Astro एक संपूर्ण वेब फ्रेमवर्क है।** इसमें वह सब कुछ शामिल है जो आपको एक वेबसाइट बनाने के लिए आवश्यक है, अंतर्निहित। किसी परियोजना को आपके सटीक उपयोग के मामले और जरूरतों के अनुसार अनुकूलित करने के लिए सैकड़ों [एकीकरण](https://astro.build/integrations/) और [API hooks](/hi/reference/integrations-reference/) भी उपलब्ध हैं।
```
Comment (Mayank-Malviya) [inline src/content/docs/hi/concepts/why-astro.mdx:11]:
```suggestion
**Astro एक ऑल - इन - वन वेब फ्रेमवर्क है।** इसमें वह सब कुछ शामिल है जो आपको एक वेबसाइट बनाने के लिए आवश्यक है, अंतर्निहित। किसी परियोजना को आपके सटीक उपयोग के मामले और जरूरतों के अनुसार अनुकूलित करने के लिए सैकड़ों [एकीकरण](https://astro.build/integrations/) और [API hooks](/hi/reference/integrations-reference/) भी उपलब्ध हैं।
```

File: src/content/docs/hi/concepts/why-astro.mdx [hunk 3]
```diff
@@ -1,45 +1,57 @@
...
+
+Astro का सर्वर-प्रथम दृष्टिकोण आपको केवल तभी क्लाइंट-साइड रेंडरिंग का विकल्प चुनने की अनुमति देता है, जब बिल्कुल आवश्यक हो। आप क्लाइंट पर चलने वाले UI फ्रेमवर्क अवयवों को जोड़ना चुन सकते हैं। चुनिंदा पेज ट्रांज़िशन्स और एनिमेशन पर बेहतर नियंत्रण के लिए आप Astro के व्यू ट्रांज़िशन्स रूटर का लाभ उठा सकते हैं। Astro का सर्वर-प्रथम रेंडरिंग, या तो प्री-रेंडर या ऑन-डिमांड, परफॉर्मेंट डिफॉल्ट प्रदान करता है जिसे आप बढ़ा और बढ़ा सकते हैं।
 
 ## डिफ़ॉल्ट रूप से तेज़
 
-अच्छा प्रदर्शन हमेशा महत्वपूर्ण होता है, लेकिन सामग्री-केंद्रित वेबसाइटों के लिए यह विशेष रूप से महत्वपूर्ण है। यह अच्छी तरह से सिद्ध हो चुका है कि खराब प्रदर्शन से आपकी सहभागिता, रूपांतरण और धनराशि कम होती है। उदाहरण के लिए:
+अच्छा प्रदर्शन हमेशा महत्वपूर्ण होता है, लेकिन यह उन वेबसाइटों के लिए *विशेष रूप से* महत्वपूर्ण है जिनकी सफलता आपकी सामग्री प्रदर्शित करने पर निर्भर करती है। यह अच्छी तरह से सिद्ध हो चुका है कि खराब प्रदर्शन से आपकी सहभागिता, रूपांतरण और पैसा ख़त्म हो जाता है। उदाहरण के लिए:
```
Comment (Mayank-Malviya) [inline src/content/docs/hi/concepts/why-astro.mdx:54]: *can be impoved*

**Review by Mayank-Malviya — COMMENTED** [review]
Couple of suggestions. Feel free to use them.

**Review by Mayank-Malviya — APPROVED** [review]
Seems good

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Amazing work 🚀

**Pull request merged**